db/db_impl/db_impl.cc

```diff
@@ -1311,7 +1235,7 @@ Status DBImpl::SetOptions(
       // Trigger possible flush/compactions. This has to be before we persist
       // options to file, otherwise there will be a deadlock with writer
       // thread.
-      InstallSuperVersionAndScheduleWork(cfd, &sv_context);
+      InstallSuperVersionForConfigChange(cfd, &sv_context);
```

Comment:
The LogAndApply() call above can release mutex_, can a flush happen during that time and installs a SV but with empty SeqnoToTimeMapping?

Reply:
Thanks! Should be fixed from #13384